content/security/committers.md

```diff
@@ -134,14 +139,14 @@ not have a dedicated security list)
 
     d. `oss-security@lists.openwall.com` ([subscription not required](http://oss-security.openwall.org/wiki/mailing-lists)).
 
+This is the first point that any information regarding the vulnerability is made public.
+
 ### Complete
 
 17. The project team updates the project's security pages.
 
-18. Set the status of the vulerability to 'READY' in the internal portal. This notifies the
-    security team, which will submit the information to the CVE project.
-
-    This is the first point that any information regarding the vulnerability is made public.
+18. Add the link to the public announcement on the mailinglist as a 'reference' of type `CONFIRM` in the CVE, and set the status of the vulerability to `READY`.
```

Comment:
I've updated the page to reflect the process as we execute it right now.

I'm not sure about asking projects to move the issue to `PUBLIC` themselves: long-term it might make sense to make the transition from `DRAFT` to `READY` be the hand-off from the project to the security team, after which we do a final review (if not already done), and then the final tasks of sending the emails (if not manually done), pushing to Mitre and moving to `PUBLIC` could be automatic?